GameOfLifeApp/Program.cs

```diff
@@ -1,2 +1,12 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using GameOfLifeApp;
+
+Console.Title = $"Game Of Life.";
+
+var gameManager = new GameManager(new Communicator(), new Display(), new GameLogic(), new Serializer());
+
+IApplicationManager running = new ApplicationManager(
+    new Communicator(),
+    new Serializer(),
+    gameManager);
+
```

Comment:
you have two Communicator and Serializer, please update to use one object.

Reply:
Done.